timestamp-impl/src/test/java/com/palantir/timestamp/InMemoryTimestampBoundStore.java

```diff
@@ -28,6 +30,7 @@ public long getUpperLimit() {
         return upperLimit;
     }
 
+    @SuppressWarnings("NonAtomicVolatileUpdate")
```

Comment:
Why not make `numberOfAllocations` atomic too?

Reply:
it is! but you're right, this line shouldn't be there